views/home.hbs

```diff
@@ -54,4 +64,22 @@
                 {{/each}}
 
             </div>
-    </section>
\ No newline at end of file
+    </section>
+
+
+
+   <script>
+            var counterContainer = document.querySelector(".displayNum");
+            
+            var visitCount = localStorage.getItem("page_view");
+    
+            // Check if page_view entry is present
+            if (visitCount) {
+            visitCount = Number(visitCount) + 1;
+            localStorage.setItem("page_view", visitCount);
```

Comment:
- Here, the `page_view` is updated only in the Local Storage, and not in the DB. Hence, the counter displays our own reload count and not the **Website View Count**.
- - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - 
- It'll be even better if we could display **"Unique Visitor Count"** along with **Website View Count**.